.gitmodules

```diff
@@ -27,3 +27,6 @@
 [submodule "test/xrt/cxxopts"]
 	path = test/xrt/cxxopts
 	url = https://github.com/jarro2783/cxxopts.git
+[submodule "jsoncpp"]
```

Comment:
Already a dependency of the project, need to manually compile it if old ABI is used.